src/inventory_client/inventory_client.go

```diff
@@ -291,9 +291,9 @@ func (c *inventoryClient) createDownloadParams(filename string) *installer.Downl
 	}
 }
 
-func (c *inventoryClient) createUpdateHostInstallProgressParams(hostId string, newStage models.HostStage, info string) *installer.UpdateHostInstallProgressParams {
+func (c *inventoryClient) createUpdateHostInstallProgressParams(infraEnvId, hostId string, newStage models.HostStage, info string) *installer.UpdateHostInstallProgressParams {
 	return &installer.UpdateHostInstallProgressParams{
-		ClusterID: c.clusterId,
+		ClusterID: strfmt.UUID(infraEnvId),
```

Comment:
same, I think that we should have an API that accepts InfraEnvID and then use that API

Reply:
i think we will start moving to that API once we have late binding working, there will be quite a lot of them

Reply:
but then you will need to come back here and change again the code that you are just putting now,
I'm not sure how big is the task of duplicating host APIs (with the definitions as they are) from v1 to v2

Reply:
changing UpdateHostInstallProgress to V2